onnxscript/ir/passes/__init__.py

```diff
@@ -0,0 +1,27 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+__all__ = [
```

Comment:
nit: It'd better reorder them in alphabetical order.

Reply:
Done